crates/bevy_ecs/src/change_detection.rs

```diff
@@ -29,10 +25,13 @@ pub trait DetectChanges {
     /// system.
     fn is_added(&self) -> bool;
 
-    /// Returns true if (and only if) this value been changed since the last execution of this
-    /// system.
+    /// Returns true if (and only if) this value has been mutably accesses since the last execution
```

Comment:
```suggestion
    /// Returns true if (and only if) this value has been mutably accessed since the last execution
```

Reply:
This is not actually correct: it also returns true on resource addition.

Reply:
What would you propose as a better alternative?
These were just copied and pasted from the previous docs or results of review comments.

Reply:
If this value has been initialized or mutably accessed since the last execution